packages/web-components/src/dialog-body/dialog-body.styles.ts

```diff
@@ -0,0 +1,124 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '../utils/index.js';
+import {
+  colorNeutralBackground1,
+  colorNeutralForeground1,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontSizeBase500,
+  fontWeightRegular,
+  fontWeightSemibold,
+  lineHeightBase300,
+  lineHeightBase500,
+  spacingHorizontalS,
+  spacingHorizontalXXL,
+  spacingVerticalL,
+  spacingVerticalS,
+  spacingVerticalXXL,
+} from '../theme/design-tokens.js';
+
+/** Dialog Body styles
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+
+  /* formerly .root */
+  :host {
+    background: ${colorNeutralBackground1};
+    box-sizing: border-box;
+    display: grid;
+    gap: ${spacingVerticalS};
+    padding: ${spacingVerticalXXL} ${spacingHorizontalXXL};
+    container: dialog-body / inline-size;
+  }
+
+  .title {
+    font-size: ${fontSizeBase500};
+    line-height: ${lineHeightBase500};
+    font-weight: ${fontWeightSemibold};
+    font-family: ${fontFamilyBase};
+    color: ${colorNeutralForeground1};
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    column-gap: 8px;
+
+    /* Sticky header */
```

Comment:
Do we need this comment?

Reply:
Comment removed! This actually spurned me to refactor the CSS and I was able to remove quite a bit of extra CSS. 🙌